apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState } from "react";
+import { View } from "react-native";
+
+import { SendNoteButton } from "./SendNoteButton";
+import { useAvailMessagingApps } from "../../../logic/messagingApps";
+import { SendAmountChooser } from "../../shared/AmountInput";
+import Spacer from "../../shared/Spacer";
+import { TextCenter, TextLight } from "../../shared/text";
+
+export function SendNoteTab() {
+  // Create Note shows available secure messaging apps
```

Comment:
nit: `Create` -> `Send`?

Reply:
for sure.

i might rename everything to PaymentLink (the contract can still be `EphemeralNotes`, but won't use "notes" anywhere else)

Reply:
fixed